pkg/destroy/bootstrap/bootstrap.go

```diff
@@ -30,17 +30,6 @@ func Destroy(dir string) (err error) {
 	tfPlatformVarsFileName := fmt.Sprintf(cluster.TfPlatformVarsFileName, platform)
 	copyNames := []string{terraform.StateFileName, cluster.TfVarsFileName, tfPlatformVarsFileName}
```

Comment:
Let's revert the addition of the copyNames variable that commit 5e285cc added - we don't need it anymore

Reply:
Looks good to me too, apart from what @markmc pointed out.

Reply:
Great catch... done.